internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -192,6 +197,36 @@ func (e *EnvStackConfig) Parameters() ([]*cloudformation.Parameter, error) {
 	return e.transformParameters(currParams, e.prevParams, transformEnvControllerParameters)
 }
 
+func getSecurityGroupConfig(mft *manifest.Environment) (*template.SecurityGroupConfig, error) {
```

Comment:
should this be in the `stack/transformers.go` file? and renamed `convertEnvSecurityGroupCfg` to be consistent with the rest of package